manager/integration/tests/run.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
 
-if [[ $@ =~ "--junitxml=" ]] ; then 
-  pytest -v "$@" > /tmp/longhorn-pytest 2>&1
-  
-  cat ${LONGHORN_JUNIT_REPORT_PATH}
+if [[ $@ =~ "--junitxml=" ]] ; then
```

Comment:
The implementation of this script is a little confusing because both conditions are almost the same except set option & flake8, and actually flake8 here seems not quite useful.

I would suggest have `pytest -v "$@" here to simplify it.